packages/flutter_build/pubspec.yaml

```diff
@@ -0,0 +1,55 @@
+name: flutter_build
+
+environment:
+  # The pub client defaults to an <2.0.0 sdk constraint which we need to explicitly overwrite.
+  sdk: ">=2.0.0-dev.68.0 <3.0.0"
+
+dependencies:
+  # To update these, use "flutter update-packages --force-upgrade".
```

Comment:
Should we add `flutter_build` here?

https://github.com/flutter/flutter/blob/d0595f6bbd460c0835a404307a5252758c5fb795/packages/flutter_tools/lib/src/commands/update_packages.dart#L122

Reply:
Done